src/components/deviceMap/deviceMap.scss

```diff
@@ -12,10 +17,37 @@
     position: relative;
     overflow: hidden;
 
-    .static-bing-map { height: 336px; }
-  }
+    .static-bing-map { height: 356px; }
+
+    .static-map-modal {
+      margin-left: auto;
+      margin-right: auto;
+      width: 50%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      top: 0px;
+      left: 0px;
+      right: 0px;
+      position: absolute;
+      bottom: 0px;
+      box-shadow: 0px 16px 72px 0px rgba(0,0,0,0.3);

```

Comment:
The modal is not correctly centered.  If you look at the map the box shadow is being dropped in strange places.

Reply:
Actually Box shadow is removed because it is causing issue in Edge and chrome as well.